### PR TITLE
PWX-36985: Modifying events in action CR for DR workflow.

### PR DESCRIPTION
**Pull request by diptiranjanpx**
**What type of PR is this?**
cleanup

**What this PR does / why we need it**:
Improving the messaging of the events in DR workflow with action CR.

**Does this PR change a user-facing CRD or CLI?**:
no

**Is a release note needed?**:
no

**Does this change need to be cherry-picked to a release branch?**:
yes, 24.2.0

### CODE REVIEW

File: pkg/action/drutils.go [hunk 1]
```diff
@@ -382,7 +382,7 @@ func (ac *ActionController) waitAfterScaleDown(action *storkv1.Action) {
 
 	// if no migration found, abort the failback process
 	if latestMigration == nil {
-		msg := fmt.Sprintf("No migration found for MigrationSchedule %s, hence aborting %s operation", referencedMigrationScheduleForLatestMigration.Name, action.Spec.ActionType)
+		msg := fmt.Sprintf("No migration detected for MigrationSchedule %s, resulting in the abortion of the %s operation.", referencedMigrationScheduleForLatestMigration.Name, action.Spec.ActionType)
```
Comment (strivedi-px) [inline pkg/action/drutils.go:385]:
```suggestion
		msg := fmt.Sprintf("No migration detected for MigrationSchedule %s/%s, resulting in the abortion of the %s operation.", referencedMigrationScheduleForLatestMigration.Namespace, referencedMigrationScheduleForLatestMigration.Name, action.Spec.ActionType)
```

File: pkg/action/drutils.go [hunk 2]
```diff
@@ -715,12 +715,12 @@ func (ac *ActionController) isClusterAccessible(action *storkv1.Action, config *
 			time.Sleep(waitInterval)
 			continue
 		}
-		msg := fmt.Sprintf("Cluster Accessibility test passed. K8s version of the cluster : %s is %v", config.Host, k8sVersion.String())
+		msg := fmt.Sprintf("Cluster accessibility test succeeded. Kubernetes version of cluster %s is %v", config.Host, k8sVersion.String())
```
Comment (strivedi-px) [inline pkg/action/drutils.go:718]:
```suggestion
		msg := fmt.Sprintf("Cluster accessibility test succeeded. Kubernetes version of cluster %s is %s", config.Host, k8sVersion.String())
```

File: pkg/action/failback.go [hunk 3]
```diff
@@ -54,7 +54,7 @@ func (ac *ActionController) verifyMigrationScheduleBeforeFailback(action *storkv
 		}
 
 		if isSubList, _, _ := utils.IsSubList(action.Spec.ActionParameter.FailbackParameter.FailbackNamespaces, migrationNamespaces); !isSubList {
-			msg := fmt.Sprintf("Namespaces provided for failback is not a subset of namespaces from MigrationSchedule %s", migrationSchedule.Name)
+			msg := fmt.Sprintf("The namespaces specified for failback are not contained within the namespaces listed in MigrationSchedule %s", migrationSchedule.Name)
```
Comment (strivedi-px) [inline pkg/action/failback.go:57]:
```suggestion
			msg := fmt.Sprintf("The namespaces specified for failback are not contained within the namespaces listed in MigrationSchedule %s/%s", migrationSchedule.Namespace, migrationSchedule.Name)
```

File: pkg/action/failback.go [hunk 4]
```diff
@@ -70,7 +70,7 @@ func (ac *ActionController) verifyMigrationScheduleBeforeFailback(action *storkv
 
 	// if no migration found, abort the failback process
 	if latestMigration == nil {
-		msg := fmt.Sprintf("No migration found for MigrationSchedule %s, hence aborting failback", migrationSchedule.Name)
+		msg := fmt.Sprintf("No migration detected for MigrationSchedule %s, resulting in the abortion of the failback operation.", migrationSchedule.Name)
```
Comment (strivedi-px) [inline pkg/action/failback.go:73]:
```suggestion
		msg := fmt.Sprintf("No migration detected for MigrationSchedule %s/%s, resulting in the abortion of the failback operation.", migrationSchedule.Namespace, migrationSchedule.Name)
```

File: pkg/action/failback.go [hunk 5]
```diff
@@ -125,7 +125,7 @@ func (ac *ActionController) verifyMigrationScheduleBeforeFailback(action *storkv
 			if !*migrationSchedule.Spec.Suspend {
 				migrationSchedule.Spec.Suspend = &suspend
 			}
-			msg := fmt.Sprintf("Suspending migration schedule %s before doing failback operation", migrationSchedule.Name)
+			msg := fmt.Sprintf("MigrationSchedule %s is being suspended prior to executing the failback operation", migrationSchedule.Name)
```
Comment (strivedi-px) [inline pkg/action/failback.go:128]:
```suggestion
			msg := fmt.Sprintf("MigrationSchedule %s/%s is being suspended prior to executing the failback operation", migrationSchedule.Namespace, migrationSchedule.Name)
```

**Review by strivedi-px — APPROVED** [review]
Looks good. Added some minor nit suggestions.

File: pkg/action/drutils.go (hunk 1 above)
Comment (diptiranjanpx) [reply pkg/action/drutils.go]: Done

File: pkg/action/failback.go (hunk 4 above)
Comment (diptiranjanpx) [reply pkg/action/failback.go]: Done

File: pkg/action/failback.go (hunk 5 above)
Comment (diptiranjanpx) [reply pkg/action/failback.go]: Done

**Pull request merged**